ws/mocks/things.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Mainflux
```

Comment:
If this is exactly the same as the HTTP adapter mock, you can use that one. Later, we should move mocks to the respective services, rather than copying them.

Reply:
Yes, it's the same; I'll import the HTTP one and delete this file.